docs/guide/structure-applications.md

```diff
@@ -330,15 +330,15 @@ image size as a parameter like the following:
 ]
 ```
 
-Then in your code where you need to use the size value, you can simply use the code like the following:
+Then in your code where you need to use the size value, you can simply use code like the following:
```

Comment:
Yeah, I waffled about this instance--I was initially going to change it to match the others, but it felt to me like it read better this way.